### DISCUSSION TITLE
Unable to start 5.7.0 on Windows

### DISCUSSION THREAD

**Discussion by pavel-faltynek** [warmuuh/milkman#185] [General]
Is there something missing?
In `5.6.0`, there seems to be also embedded runtime in `jre-win64`, also as the `exe` file (contrary to `5.7.0`).

Is there any way, how to get all existing plugins and install them? (I would prefer the "binary" way, don't want to mess up with java stuff).

Thanks for any info.

**Comment by warmuuh** [comment]
I'll check. Might be something wrong with the release process :/

**Reply by warmuuh** [reply]
fixed: https://github.com/warmuuh/milkman/releases/tag/5.7.1
you can also install via choco (it takes a while to catch up though), then, i think, you'll get all the plugins as well)

**Reply by pavel-faltynek** [reply]
🆒 , thank you
